packages/design-system-tokens/src/themes/medicare.ts

```diff
@@ -104,29 +104,43 @@ export const shadow: ShadowTokens = {
   'focus':                      `inset 0 0 0 1px ${themeColors['base']}`,
   'focus-inverse':              `inset 0 0 0 1px ${themeColors['base']}`,
   'focus-link':                 `0 3px ${themeColors['base']}`,
-  'box-card': '0 2px 3px 0 rgba(50, 50, 50, 0.23)',
+  'box-card':                   '0 2px 3px 0 rgba(50, 50, 50, 0.23)',
   'base-offset-x':              '2px',
   'base-offset-y':              '2px',
   'base-blur-radius':           '4px',
   'base-color':                 color['black-alpha25'],
   'base':                       '2px 2px 4px',
 };
 
-const fonts: FontTokens = {
-  ...font,
-  'size-lg': '1.25rem',
-  'size-xl': '1.5rem',
-  'size-2xl': '2rem',
-  'size-3xl': '2.5rem',
-  'size-4xl': '2.75rem',
-  'size-5xl': '3.125rem',
-  sans: font['family-rubik'],
-  montserrat: font['family-montserrat'],
-  rubik: font['family-rubik'],
-  'weight-light': 400,
-  'weight-normal': 400,
-  'weight-semibold': 600,
-  'weight-bold': 600,
+const font: FontTokens = {
+  'sans':                       fontFamily['family-rubik'],
+  'montserrat':                 fontFamily['family-montserrat'],
+  'rubik':                      fontFamily['family-rubik'],
+  'size-base':                  fontSize['20'],
+  'size-sm':                    fontSize['10'],
+  'size-md':                    fontSize['20'],
+  'size-lg':                    '1.25rem',
+  'size-xl':                    '1.5rem',
```

Comment:
should these sizes be called out as "--desktop" since we're making a distinction between mobile and tablet? i'll point out what i mean further down with a 🍕